src/main/java/br/com/budgetboys/pf_finance_service/adapters/inbound/controller/expense/ExpenseController.java

```diff
@@ -0,0 +1,55 @@
+package br.com.budgetboys.pf_finance_service.adapters.inbound.controller.expense;
+
+import br.com.budgetboys.pf_finance_service.adapters.outbound.service.expense.ExpenseService;
+import br.com.budgetboys.pf_finance_service.domain.expense.ExpenseCreateDTO;
+import br.com.budgetboys.pf_finance_service.domain.expense.ExpenseResponseDTO;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("expense")
```

Comment:
Transformar rota para o padrão RESTful
```suggestion
@RequestMapping("api/expenses")
```

Reply:
O padrão está como finance/expense, ainda é necessário colocar o api/?